pom.xml

```diff
@@ -267,6 +267,13 @@
             <version>1.9.3</version>
             <scope>test</scope>
         </dependency>
+        <!-- https://mvnrepository.com/artifact/com.nhaarman.mockitokotlin2/mockito-kotlin -->
+        <dependency>
+            <groupId>com.nhaarman.mockitokotlin2</groupId>
```

Comment:
I can review this PR tomorrow, but can we stick to one test framework and not introduce another one?

Reply:
yes, I forgot to update it to mockk, sorry!